### DISCUSSION TITLE
ImportError: cannot import name 'ModelField' from 'pydantic.fields' (C:\Users\wyhma\wyhmaster\.venv\Lib\site-packages\pydantic\fields.py)

### DISCUSSION THREAD

**Discussion by wyhmaster** [nonebot/discussions#543] [问答]
部署bot后，出现 ImportError: cannot import name 'ModelField' from 'pydantic.fields' (C:\Users\wyhma\wyhmaster\.venv\Lib\site-packages\pydantic\fields.py)这一行错误，部署操作正确。
![Uploading 89C2989A709BE661395C7E6649ABDD05.png…]()

**Comment by yanyongyu** [comment]
你报错的图片上传失败了，另外你的 nonebot2 版本是？

**Comment by wyhmaster** [comment]
nonebot 1.4.2 已确认更新



---Original---
From: ***@***.***&gt;
Date: Sun, Dec 15, 2024 18:07 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nonebot/discussions] ImportError: cannot import name'ModelField' from 'pydantic.fields'(C:\Users\wyhma\wyhmaster\.venv\Lib\site-packages\pydantic\fields.py)(Discussion #543)




 
你报错的图片上传失败了，另外你的 nonebot2 版本是？

**Reply by yanyongyu** [reply]
nonebot2 没有 1.4.2 这个版本。。。你看的是nb-cli吧，`pip show nonebot2`。另外请提供你的报错内容。

**Comment by wyhmaster** [comment]
找到了，2.3.3



---Original---
From: ***@***.***&gt;
Date: Sun, Dec 15, 2024 21:21 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nonebot/discussions] ImportError: cannot import name'ModelField' from 'pydantic.fields'(C:\Users\wyhma\wyhmaster\.venv\Lib\site-packages\pydantic\fields.py)(Discussion #543)




 
nonebot2 没有 1.4.2 这个版本。。。你看的是nb-cli吧，pip show nonebot2

**Reply by yanyongyu** [reply]
那就不是nb的问题了，看你报错是谁导致的

**Comment by wyhmaster** [comment]
不知道怎么解决了，重新部署了一次bot，再打开就不报错了



---Original---
From: ***@***.***&gt;
Date: Sun, Dec 15, 2024 22:56 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nonebot/discussions] ImportError: cannot import name'ModelField' from 'pydantic.fields'(C:\Users\wyhma\wyhmaster\.venv\Lib\site-packages\pydantic\fields.py)(Discussion #543)




 
那就不是nb的问题了，看你报错是谁导致的
